doc/mgr/orchestrator_cli.rst

```diff
@@ -105,20 +118,43 @@ filtered to a particular node:
 
 ::
 
-    ceph orchestrator device ls [--host=...] [--devname=...] [--refresh]
+    ceph orchestrator device ls [--host=...] [--refresh]
 
 Create OSDs
 ^^^^^^^^^^^
 
 Create OSDs on a group of devices on a single host::
 
-    ceph orchestrator osd create <host> <ceph-volume-invocation…>
+    ceph orchestrator osd create [flags...] <host>:<drive>
```

Comment:
I think we decided to use only one parameter "drive-group", with the same fields that we have in the json file.

Reply:
I do not like to have a big number of parameters in the command line, but is the only possibility if we cannot use a json string.... ( why is not acceptable to have it?)

Reply:
accepting JSON on the command line is really unusual. JSON data structures are typically read from stdin or from a file.

Reply:
Ok. Let's put the different optional flags here and leave the "-i file" to read a json drive group structure.